rocky/rocky/views/organization_add.py

```diff
@@ -33,13 +39,30 @@ def get_context_data(self, **kwargs):
         return context
 
     def form_valid(self, form):
-        self.object = form.save()
-        member, _ = OrganizationMember.objects.get_or_create(user=self.request.user, organization=self.object)
-        member.acknowledged_clearance_level = 4
-        member.trusted_clearance_level = 4
-        member.save()
+        try:
+            self.object = form.save()
+        except (OctopoesException, KatalogusException) as e:
+            service = "Octopoes" if isinstance(e, OctopoesException) else "the Katalogus"
```

Comment:
Woudn't it be more logical to have the service available as a value on the exception class?

Reply:
Yes or:
```python3
except OctopoesException:
    # show specific error on octopoes
except KatalogusException:
    # show specific error on KATalogus
else:
    # no exceptions, redirect
```

Reply:
I also considered this, but was a bit hesitant as that would add representation data in the exception.

Reply:
I added a base class called `ServiceException` to represent issues with (external) services and carries the name.